### PR TITLE
change to environment_ids

### PR DESCRIPTION
**Pull request by swadeley**
due to API change

Hi, while testing I found this change had not been made when API changed a few months ago.

### CODE REVIEW

File: tests/upgrades/test_client.py [hunk 1]
```diff
@@ -110,7 +110,7 @@ def create_activation_key_for_client_registration(ak_name, client_os, org, envir
     tools_cv = tools_cv.read()  # Published CV with new version
     # Promote CV
     cvv = entities.ContentViewVersion(id=max([cvv.id for cvv in tools_cv.version])).read()
-    cvv.promote(data={'environment_id': environment.id, 'force': False})
+    cvv.promote(data={'environment_ids': environment.id, 'force': False})
```
Comment (jyejare) [inline tests/upgrades/test_client.py:113]: You sure that parameter (`environment_ids`) takes integer and not list ?

Comment (jyejare) [reply tests/upgrades/test_client.py:113]: Also, you should cherrypick this in 6.9.z once merged !

Comment (swadeley) [reply tests/upgrades/test_client.py:113]: @jyejare I am of course happy to submit test results or just add [ ] if you think best.

I made the change based on my change here https://github.com/SatelliteQE/robottelo/pull/8001 on 17 Sep 2020
and test runs after this change.
BTW, I see I did ping @devendra104  at the time https://github.com/SatelliteQE/airgun/issues/520#issuecomment-694249859

**Review by JacobCallahan — CHANGES_REQUESTED** [review]
ACK. Merge pending test results.

**Comment by swadeley** [comment]
Hello

test fails line 242 which is after the changes:

robottelo]$ pytest   --show-capture=no tests/upgrades/test_client.py -k test_pre_scenario_preclient_package_installation -m pre_upgrade
================================== test session starts ==========================================
platform linux -- Python 3.9.4, pytest-6.2.2, py-1.10.0, pluggy-0.13.1
rootdir: /sat6/robottelo_1/robottelo, configfile: pyproject.toml
plugins: xdist-2.1.0, mock-3.3.1, forked-1.3.0, ibutsu-1.15, services-2.2.1, cov-2.11.1, reportportal-5.0.7
collected 3 items / 2 deselected / 1 selected                                                                                                                                                     

tests/upgrades/test_client.py F                                                                                                                                                             [100%]

============================================================================================ FAILURES =============================================================================================
______________________________________________ Scenario_upgrade_old_client_and_package_installation.test_pre_scenario_preclient_package_installation ______________________________________________

self = <tests.upgrades.test_client.Scenario_upgrade_old_client_and_package_installation testMethod=test_pre_scenario_preclient_package_installation>

    @pre_upgrade
    def test_pre_scenario_preclient_package_installation(self):
        """Create product and repo from which the package will be installed
        post upgrade
    
        :id: preupgrade-eedab638-fdc9-41fa-bc81-75dd2790f7be
    
        :steps:
    
            1. Create a content host with existing client ak
            2. Create and sync repo from which the package will be
                installed on content host
            3. Add repo to CV and then in Activation key
    
        :expectedresults:
    
            1. The content host is created
            2. The new repo and its product has been added to ak using which
                the content host is created
    
        """
        prior_env = entities.LifecycleEnvironment(organization=self.org).search(
            query={'search': 'name=Library'}
        )[0]
        environment = entities.LifecycleEnvironment(
            organization=self.org, prior=prior_env.id, label=self.le_lable, name=self.le_name
        ).create()
        ak = create_activation_key_for_client_registration(
            ak_name=self.ak_name,
            client_os='rhel7',
            org=self.org,
            environment=environment,
            sat_state='pre',
        )
>       rhel7_client = dockerize(ak_name=ak.name, distro='rhel7', org_label=self.org.label)

tests/upgrades/test_client.py:242:

**Review by devendra104 — APPROVED** [review]
ACK

**Comment by swadeley** [comment]
Thank you

**Pull request merged**